2.2_comb_logic.ipynb

```diff
@@ -231,7 +231,7 @@
    "source": [
     "Notice how the Verilog contains constants instead of actual mux or concatenation logic. This is because FIRRTL transformations have simplified the circuit, eliminating obvious logic. \n",
     "\n",
-    "For a more complete list of Chisel operators, see the [Chisel cheatsheet](https://chisel.eecs.berkeley.edu/documentation.html). For the most complete list of operators and their implementation details, look through the [Chisel API](https://chisel.eecs.berkeley.edu/api/#package)."
+    "For a more complete list of Chisel operators, see the [Chisel cheatsheet](https://github.com/freechipsproject/chisel-cheatsheet/releases/latest/download/chisel_cheatsheet.pdf). For the most complete list of operators and their implementation details, look through the [Chisel API](https://chisel-lang.org/api/latest/)."
```

Comment:
Cheatsheet is not available on the website?

Reply:
It's not hosted on the website, but it is linked to under [`chisel3/resources`](https://www.chisel-lang.org/chisel3/resources).